crates/bdk/src/wallet/mod.rs

```diff
@@ -1949,6 +1949,18 @@ impl<D> Wallet<D> {
         Ok(())
     }
 
+    /// Stages wallet changes in memory for later commit.
+    ///
+    /// To commit changes to the persistence backend, call [`commit`].
+    ///
+    /// [`commit`]: Self::commit
+    pub fn stage(&mut self, changeset: ChangeSet)
```

Comment:
I think it's better not to have this. Just call `self.persist.stage` at the end of method definitions.